lake/index/query.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/segmentio/ksuid"
 )
 
+// get rid of this
```

Comment:
Looks like you can delete this file.

Reply:
Yes, this dates to zar.  We'll add a meta-query way to scan the index-pruned data objects list for test and debug...